r_markdown/carbonyls.Rmd

```diff
@@ -123,7 +123,8 @@ Load notes
                                     -id, -id_ions, -flow,
                                     -start, -end, -type, -qc) %>%
                       dplyr::filter(!is.na(pol)) %>%
-                      dplyr::mutate(pol = as.factor(pol))
+                      dplyr::mutate(pol = as.factor(pol)) %>%
+                      dplyr::select(-id_ions)
```

Comment:
"id_ions" comes from the excel file, its used because that's how the data was recorded then dropped